src/Middleware/NodeServices/test/NodeServicesTest.cs

```diff
@@ -4,12 +4,14 @@
 using System;
 using System.IO;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
 using Microsoft.AspNetCore.NodeServices.HostingModels;
 using Microsoft.Extensions.DependencyInjection;
 using Xunit;
 
 namespace Microsoft.AspNetCore.NodeServices
 {
+    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/22084", Queues = "Windows.10.Arm64;Windows.10.Arm64.Open")]
```

Comment:
Wow, a lot of tests need to be skipped on the ARM64 queues. Are major product issues seen on this platform?

Reply:
Yeah nodejs isn't officially supported on win arm64